packages/manager/apps/hub/src/dashboard/dashboard.controller.js

```diff
@@ -1,7 +1,43 @@
+import {
+  BANNER_IP_REPRICING_AVAILABILITY,
+  TRACKING_IMPRESSION_DATA,
+} from './dashboard-contants';
+
 export default class DashboardController {
+  /* @ngInject */
+  constructor(atInternet, hubDashboardService, ovhFeatureFlipping) {
+    this.atInternet = atInternet;
+    this.hubDashboardService = hubDashboardService;
+    this.ovhFeatureFlipping = ovhFeatureFlipping;
+  }
+
   $onInit() {
+    this.ovhFeatureFlipping
```

Comment:
**question:** Shouldn't we wrap all async calls in a `$q.all` as we are on a `$onInit` hook?

Reply:
Its's applied.